doc/man3/OSSL_CMP_SRV_CTX_new.pod

```diff
@@ -128,6 +128,11 @@ confirmation of newly enrolled certificates if requested.
 
 CMP is defined in RFC 4210 (and CRMF in RFC 4211).
 
+=head1 BUGS
+
+So far the CMP server implementation is limited to one request per CMP message
+(and consequently to at most one response component per CMP message).
```

Comment:
This isn't so much a bug as a design limitation. Probably this shouldn't be in the "BUGS" section - but just in the general description.

Reply:
Yeah, I had been looking for something like a LIMITS section, but did not find any,
so resorted to (ab-)using BUGS, which apparently had already done occasionally for other types of limitations.
I've just removed the BUGS section indicator, such that the text pointing out the limitation has become part of the NOTES section :)